drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/hmis_export_fetcher.rb

```diff
@@ -15,7 +15,7 @@ class HmisExportFetcher
 
     delegate :content, to: :export
 
-    def run!
+    def run!(lookback_years: 3)
```

Comment:
I'd probably just make this take a start date so you could send an arbitrary range if you wanted.
```suggestion
    def run!(start_date: 3.years.ago.to_date)
```